stardoc/private/stardoc.bzl

```diff
@@ -114,6 +120,15 @@ _common_renderer_attrs = {
         allow_single_file = [".vm"],
         mandatory = True,
     ),
+    "generate_table_of_contents": attr.bool(
```

Comment:
alternatively, rather than providing a "generate_table_of_contents" boolean, `table_of_contents_template` could instead default to nothing, and the user could provide `Label("@stardoc//stardoc:templates/markdown_tables/table_of_contents.vm")` (or their own) if they want the ToC generated

Reply:
removed generate_table_of_contents and made table_of_contents_template empty